copter/source/docs/tuning-process-instructions.rst

```diff
@@ -80,14 +202,14 @@ The first priority when tuning an multirotor aircraft is to establish a stable t
 3. If the aircraft starts to oscillate immediately abort the take off or land the aircraft.
 4. Reduce all the following parameters by 50%
 
-a. ATC_RAT_PIT_P
-b. ATC_RAT_PIT_I
-c. ATC_RAT_PIT_D
-d. ATC_RAT_RLL_P
-e. ATC_RAT_RLL_I
-f. ATC_RAT_RLL_D
+a. :ref:`ATC_RAT_PIT_P <ATC_RAT_PIT_P__AC_AttitudeControl_Multi>`
+b. :ref:`ATC_RAT_PIT_I <ATC_RAT_PIT_I__AC_AttitudeControl_Multi>`
+c. :ref:`ATC_RAT_PIT_D <ATC_RAT_PIT_D__AC_AttitudeControl_Multi>`
+d. :ref:`ATC_RAT_RLL_P <ATC_RAT_RLL_P__AC_AttitudeControl_Multi>`
+e. :ref:`ATC_RAT_RLL_I <ATC_RAT_RLL_I__AC_AttitudeControl_Multi>`
+f. :ref:`ATC_RAT_RLL_D <ATC_RAT_RLL_D__AC_AttitudeControl_Multi>`
 
-This process is repeated until the aircraft can hover without oscillations being detectable visually or audibly.
+This process is repeated until the aircraft can hover without oscillations being detectable visually or audibly. # explain audibly ? motor sound should be constant and not noisy ?
```

Comment:
I don't think we need to explain this. When the aircraft oscillates you can generally clearly hear the pitch of the motor going up and down. This is where a video or two will help out. For now though I think this line was fine the way it was.

Reply:
could be say a chopped sound ? non constant sound ?

Reply:
I really don't think it needs any further explanation. It sounds like it looks. I am happy to wait for the video's.